templates/navbar.html

```diff
@@ -18,6 +18,13 @@
                 <a class="nav-link page-scroll" href="{% url 'home' %}">HOME</a>
             </li>
 
+            {% if user.is_authenticated %}  
+            <li class="nav-item">
+                <a class="nav-link page-scroll" href="/dashboard">DASHBOARD</a>
```

Comment:
Use [url template tag](https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#url) instead.
```suggestion
                <a class="nav-link page-scroll" href="{% url 'dashboard:dashboard' %}">DASHBOARD</a>
```